packages/victory-vendor/package.json

```diff
@@ -16,10 +16,13 @@
   "author": "Formidable",
   "license": "MIT AND ISC",
   "exports": {
+    "./package.json": "./package.json",
     "./d3-*": {
       "import": "./es/d3-*.js",
       "default": "./lib/d3-*.js"
-    }
+    },
+    "./es/d3-*": "./es/d3-*.js",
```

Comment:
If I understand correctly, we're building + distributing the `d3-*` libraries because they ONLY support ESM, and Victory needs to support CommonJS, right?
But if the Victory consumer supports ESM, we'll pass-through the raw `d3-*` libraries as dependencies.

So my question is, why not just supply the bundled `d3-*` libraries for both, and remove `d3-*` as dependencies?  (as a way to reduce download size)

Reply:
It will actually increase download size (for victory download size, would be a wash overall) to copy out all the ESM files and remove from `package.json:dependencies`. And for maintenance reasons, I would offer it's preferable to use ye olde `package.json:dependencies` to gate that.